vaadin-components-gwt/src/main/java/com/vaadin/components/grid/config/JSColumn.java

```diff
@@ -18,6 +18,18 @@
     void setName(String s);
 
     @JsProperty
+    String getHeaderText();
+
+    @JsProperty
+    void setHeaderText(String s);
+
+    @JsProperty
+    String getHidingToggleCaption();
```

Comment:
It's useful for columns with html (for example, button) instead of just text in header